quantum/rgb_matrix.h

```diff
@@ -101,6 +106,10 @@ void rgb_matrix_indicators(void);
 void rgb_matrix_indicators_kb(void);
 void rgb_matrix_indicators_user(void);
 
+void rgb_matrix_indicators_advanced(effect_params_t *params);
```

Comment:
I could be wrong, but I think `qmk cformat` may have messed this up:

```suggestion
void rgb_matrix_indicators_advanced(effect_params_t* params);
```

Reply:
Either alignment works, however the `PointerAlignment` specified in the .clang-format is `Right`.

Reply:
Can confirm.  cformat doesn't change this, and several other functions use the `Right` alignment already.